<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RfidProject_zx</class>
 <widget class="QMainWindow" name="RfidProject_zx">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RfidProject_zx</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>校园一卡通系统平台</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_log">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>341</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border-style: none;
	}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">        QGroupBox {

            border-style: none;

        }</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QPushButton" name="pushButton_openSerialPort">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>开启</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Port">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>12</y>
         <width>54</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>串口号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_BaudRate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>波特率：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_Port">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_BaudRate">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_closeSerialPort">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>电子钱包模块</string>
     </property>
     <widget class="QPushButton" name="pushButton_MoneyNum">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>查询余额</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addMoneyNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_addMoneyNum">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_subMoneyNum">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>扣款</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_subMoneyNum">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_openCardMoneyModule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>打开模块</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_closeCardMoneyModule">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭模块</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>寻卡操作</string>
     </property>
     <widget class="QPushButton" name="pushButton_requestCard">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>寻卡</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CardNum">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addCardUser">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>添加用户</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>门禁系统模块</string>
     </property>
     <widget class="QPushButton" name="pushButton_openDoorModule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>打开模块</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addUserDoorModule">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>添加用户</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_CardNumDoorModule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_closeDoorModule">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭模块</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_cleanlog">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>400</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>510</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>学号：201759225309</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
